apyros/metalog.py

```diff
@@ -57,7 +71,11 @@ def getLog( self, prefix ):
                 ret = line.split()[1].strip()
```

Comment:
Can we split by ":" here? That is the intention, right? Split prefix from suffix separated by ":".